lib/rpmfiles.h

```diff
@@ -396,13 +397,21 @@ rpm_loff_t rpmfilesFSize(rpmfiles fi, int ix);
 rpm_color_t rpmfilesFColor(rpmfiles fi, int ix);
 
 /** \ingroup rpmfiles
- * Return file class from file info set.
+ * return file class from file info set.
```

Comment:
Why did you change this?

Reply:
By mistake, rather obviously I hope. Fixed, thanks for spotting.